docs/superpowers/specs/2026-04-01-grafana-dashboard-design.md

```
@@ -0,0 +1,204 @@
+| `$channel_type` | Query | `label_values(one_api_relay_requests_total, channel_type)` | All | Yes |
+| `$model` | Query | `label_values(one_api_relay_requests_total, model)` | All | Yes |
+| `$username` | Query | `label_values(one_api_user_requests_total, username)` | All | Yes |
+| `$group` | Query | `label_values(one_api_user_requests_total, group)` | All | Yes |
+| `$currency` | Custom | `USD,CNY` | USD | No |
+| `$exchange_rate` | Textbox | - | 8 | No |
+
+### Currency Conversion
+
+All monetary panels use the following formula:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)